tests/encfs/encfs.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from contextlib import contextmanager
+import subprocess
+import tempfile
+
+
+class CryptSetupFileSystem:
+    DEVICE_NAME = "cryptdevice1"
+    DEVICE_NAME_PATH = f"/dev/mapper/{DEVICE_NAME}"
+
+    def _run_command(self, *args):
+        subprocess.check_call(
+            " ".join(
+                [
+                    "sudo cryptsetup --debug -v",
+                    *args,
+                ]
+            ),
+            shell=True,
+        )
+
+    def __init__(self, key_path, image_path):
+        self.key_path = key_path
+        self.image_path = image_path
+        with open(image_path, "wb") as f:
+            f.seek(64 * 1024 * 1024 - 1)
```

Comment:
Instead of doing the math, can we make this a constant?

Reply:
Up to you, I think the math acts as documentation that its 64mb